.github/ISSUE_TEMPLATE/release_template.md

```diff
@@ -44,7 +44,8 @@ assignees: ''
 - [ ] Get the image digests from the build process and make a commit and PR with
       these digests.
   - [ ] Run `contrib/release/pull-docker-manifests.sh` to fetch the image
-        digests, use the URL of the GitHub run here.
+        digests, use the run ID of the GitHub run here.
+  - [ ] Run `make -C install/kubernetes` to update the helm values & charts
```

Comment:
This is good.

Reply:
I just moved this step into the script, no need to force someone to manually run it.